### DISCUSSION TITLE
Switch default sprite and body/scale is very small

### DISCUSSION THREAD

**Discussion by Tsunder** [FakeFishGames/Barotrauma#12351] [Ideas and suggestions]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
The Switch is a very nice item but its default scale makes it the smallest interactable in the game, about half the size of the button, making it harder to click on, and also harder to see.

**Steps To Reproduce**
pic showing switch size vs button and lever, two switches fitting inside a button, and a 0.75 scale switch
![image](https://user-images.githubusercontent.com/5103517/143905128-b63af324-972b-440a-8763-47a5708d9038.png)

**Version**
0.15.13

**Additional information**
tbh lots of subs already use 0.5 switches placed on top of other items which make them double hard to click on. rage.

**Comment by Jallu** [comment]
@danielasteljoki put the ball rolling
